src/ucp/wireup/wireup.c

```diff
@@ -1639,14 +1648,22 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
     char str[32];
     ucs_queue_head_t replay_pending_queue;
     ucp_rsc_index_t dst_mds_mem[UCP_MAX_MDS];
+    int is_reconfigurable;
 
     tl_bitmap = UCS_STATIC_BITMAP_AND(*local_tl_bitmap,
                                       worker->context->tl_bitmap);
     ucs_assert(!UCS_STATIC_BITMAP_IS_ZERO(tl_bitmap));
 
     ucs_trace("ep %p: initialize lanes", ep);
     ucs_log_indent(1);
-    ucp_wireup_eps_pending_extract(ep, &replay_pending_queue);
+
+    if (!ucp_ep_has_cm_lane(ep)) {
+        ucp_ep_update_flags(ep, 0,
+                            UCP_EP_FLAG_LOCAL_CONNECTED |
+                            UCP_EP_FLAG_REMOTE_CONNECTED);
+    }
```

Comment:
why needed?

Reply:
o/w the wireup flow will not reconnect new lanes.